CHANGELOG.md

```diff
@@ -3,6 +3,47 @@
 All notable changes to this project will be documented in this file.
 This project adheres to Semantic Versioning.
 
+## [1.7.0] 2021-07-15
+
+### Added
+
+- Go 1.17 support (@aarzilli, @mknyszek)
+- Add new API and terminal command for setting watchpoints (@aarzilli)
+- Add filtering and grouping to goroutines command (@aarzilli)
+- Added support for hit count condition on breakpoints (@suzmue, @aarzilli)
+- DAP server: Handle SetVariable requests (@hyangah)
+- DAP server: Add clipboard support (@hyangah)
+
+### Fixed
+
+- DAP server: Send terminated event when disconnecting (@suzmue)
```

Comment:
This one was actually kind of great territory. Not clear from the spec if it is even needed since we do send disconnect request back to acknowledge termination in that way. This wasn't really breaking anything, but did impact comparative testing between old and new adapter. That said, there were several other shutdown/disconnect fixes as well, so how about just grouping them into a general category with multiple names (@suzmue, @polinasok).